docs/tutorials/word-tutorial.md

```diff
@@ -0,0 +1,962 @@
+
+2. Open a Git bash window, or Node.JS-enabled system prompt, and navigate to the **Start** folder of the project.
+
+3. Run the command `npm install` to install the tools and libraries listed in the package.json file. 
+
+4. Carry out the steps in [Adding Self-Signed Certificates as Trusted Root Certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md) to trust the certificate for your development computer's operating system.
+
+## Insert a range of text
+
+In this step of the tutorial, you'll programmatically test that your add-in supports the user's current version of Word, and then insert a paragraph into the document.
+
+### Code the add-in
+
+1. Open the project in your code editor.
+
+2. Open the file index.html.
+
+3. Replace the `TODO1` with the following markup:
+
+    ```html
+    <button class="ms-Button" id="insert-paragraph">Insert Paragraph</button>
+    ```
+
+4. Open the app.js file.
+
+5. Replace the `TODO1` with the following code. This code determines whether the user's version of Word supports a version of Word.js that includes all the APIs that are used in all the stages of this tutorial. In a production add-in, use the body of the conditional block to hide or disable the UI that would call unsupported APIs. This will enable the user to still use the parts of the add-in that are supported by their version of Word.
+
+    ```js
+    if (!Office.context.requirements.isSetSupported('WordApi', 1.3)) {
+        console.log('Sorry. The tutorial add-in uses Word.js APIs that are not available in your version of Office.');
+    }
+    ```
+
+6. Replace the `TODO2` with the following code:
+
+    ```js
+    $('#insert-paragraph').click(insertParagraph);
+    ```
+
+7. Replace the `TODO3` with the following code. Note:
+
+   - Your Word.js business logic will be added to the function that is passed to `Word.run`. This logic does not execute immediately. Instead, it is added to a queue of pending commands.
+
+   - The `context.sync` method sends all queued commands to Word for execution.
+
+   - The `Word.run` is followed by a `catch` block. This is a best practice that you should always follow. 
+
+    ```js
+    function insertParagraph() {
+        Word.run(function (context) {
+
+            // TODO4: Queue commands to insert a paragraph into the document.
+
+            return context.sync();
+        })
+        .catch(function (error) {
+            console.log("Error: " + error);
+            if (error instanceof OfficeExtension.Error) {
+                console.log("Debug info: " + JSON.stringify(error.debugInfo));
+            }
+        });
+    }
+    ```
+
+8. Replace `TODO4` with the following code. Note:
+
+   - The first parameter to the `insertParagraph` method is the text for the new paragraph.
+
+   - The second parameter is the location within the body where the paragraph will be inserted. Other options for insert paragraph, when the parent object is the body, are "End" and "Replace".
```

Comment:
Here and in the code block, should we use the InsertLocation enum instead of the string literal?

Reply:
Agreed; we'll plan to revisit this later when the content of this tutorial is re-evaluated / refactored.